en/docs/learn/device-flow-grant.md

```diff
@@ -0,0 +1,47 @@
+# Device Authorization Grant (Device Flow)
+
+### Recommended use
+
+This is an OAuth 2.0 extension that enables enables OAuth clients to 
+request user authorization from applications on,
+
+* Input constrained devices
+* Devices that have a lack of browser
+
+Such devices include smart TVs, printers and play-stations. This 
+authorization flow defined as the “device flow”. In this specification, 
+it instructs the user to review the authorization request on a secondary 
+device, such as a smartphone.
+
+### The flow
+
+Device flow doesn’t require two-way communication between the OAuth 
+client on the device. Instead of direct contact with end-user’s user-agent, 
+It instructs the end-user to connect the
+ authorization server through another device and approve the access request.
+ 
+The diagram below illustrates the device flow.
+
+![device-authorization-grant-diagram](../assets/img/using-wso2-identity-server/deviceflow.png)
+
+(A) — Client requests access from authorization server including its client identifier
+
```

Comment:
The client sends an access request including it's client identifier to the authorization server.